### ISSUE TITLE
No way to update the blocklist without restarting

### ISSUE THREAD

**Issue by anthonybriand** [webtorrent/webtorrent#1241]
**What version of WebTorrent?**
0.98.20
**What operating system and Node.js version?**
MacOS X 10.13 / Node.JS 8.9.1
**What did you expect to happen?**
A way to update the blocklist programmaticaly without restarting the client
**What actually happened?**
Nothing because no method is available

**Comment by DiegoRBaquero** [comment]
I'm lost. What's the blocklist?

**Comment by anthonybriand** [comment]
Sorry I wasn't clear, it's corresponding to the loaded ipSet, cf code block: https://github.com/webtorrent/webtorrent/blob/0bf7b97ae411970c69909fdf1ecdfb011652cf75/index.js#L149

It would be important to be able to update periodically or on demand the ipSet blocklist without restarting the client

**Comment by feross** [comment]
Fair point, once WebTorrent Desktop exposes this functionality in Preferences, we'll probably need this functionality.
